pystac/utils.py

```diff
@@ -337,33 +380,86 @@ def extract_coords(coords: List[Union[List[float], List[List[Any]]]]) -> None:
 
 
 def map_opt(fn: Callable[[T], U], v: Optional[T]) -> Optional[U]:
-    """Maps the value of an option to another value, returning
-    None if the input option is None.
+    """Maps the value of an optional type to another value, returning
+    ``None`` if the input option is ``None``.
+
```

Comment:
Should this have `Args:` and `Returns:`?

Reply:
Added in b47dceb3da0952cd4148e710db98318ca9b17c73